yarpp-template-unwind.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/*
+YARPP Template: SiteOrigin Unwind
+*/ ?>
+<h2 class="related-posts heading-strike"><?php esc_html_e( 'You may also like', 'siteorigin_unwind' ); ?></h2>
+<?php if (have_posts()):?>
+	<ol>
+		<?php while (have_posts()) : the_post(); ?>
+			<li><a href="<?php the_permalink() ?>" rel="bookmark" title="<?php the_title_attribute(); ?>">
+				<?php if (has_post_thumbnail()):?>
+					<?php the_post_thumbnail('related-post'); ?>
+				<?php endif; ?>
+				<h3 class="related-post-title"><?php the_title(); ?></h3>
+				<p class="related-post-date"><?php the_time( 'M d, Y' ); ?></p>
+			</a></li>
+		<?php endwhile; ?>
+	</ol>
+	<div class="clear"></div>
+
+<?php else: ?>
+	<p>No related photos.</p>
```

Comment:
Thanks for pointing it out. My bad.

I just modified one of the plugin's default templates ..